# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pstoedit/gcc43.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pstoedit-3.45/src/cppcomp.h.vanilla	2008-08-10 11:39:34.000000000 +0200
+++ pstoedit-3.45/src/cppcomp.h	2008-08-10 11:40:11.000000000 +0200
@@ -31,7 +31,8 @@
 #endif
 
 
-
+#include <string.h>
+#include <stdlib.h>
 
 #ifdef _AIX
 #define _unix
